lib/miq_automation_engine/service_models/miq_ae_service_tenant.rb

```diff
@@ -1,17 +1,5 @@
 module MiqAeMethodService
   class MiqAeServiceTenant < MiqAeServiceModelBase
-    expose :id
-    expose :domain
-    expose :subdomain
-    expose :name
```

Comment:
@kbrock The service model automatically exposes columns and virtual_columns as read-only for the models so you do not need to explicitly state them.  Relationships do need to be explicitly stated.